docs/src/pages.ts

```diff
@@ -326,7 +326,6 @@ const pages: readonly MuiPage[] = [
       { pathname: '/discover-more/roadmap' },
       { pathname: '/discover-more/backers', title: 'Sponsors & Backers' },
       { pathname: '/discover-more/vision' },
-      { pathname: '/discover-more/team' },
```

Comment:
Should we instead of removing this add a link to the About us page? cc @danilo-leal @oliviertassinari

Reply:
Or, maybe it makes sense to keep the team section, one per product, e.g. https://reactjs.org/community/team.html. We could add one for MUI X, once we split the docs.

Reply:
That would be even better. I like their short bio for each member 😄 But for now, maybe a link to the About us page would be enough. Feels weird to have it only on the landing page.

Reply:
The team section on the About page is company-wide, which works for now. Whenever and if we split the docs, having one per product is indeed a nice thing because people can reach out to the specific team members that maintain Core, X, and so on. :)

Reply:
Alright, will add it and leave this open for future reference

Reply:
SO to be clear, the plan is to:

1. remove the duplication (this PR)
2. add the team pages back once its clearer that we have different products, one per team 👍